app/config.py

```diff
@@ -134,6 +134,9 @@ class Config(object):
     SECRET_KEY = os.getenv("SECRET_KEY")
     DANGEROUS_SALT = os.getenv("DANGEROUS_SALT")
 
+    # API key prefix
```

Comment:
API is the source of truth for the prefix value.  This is a static value across all environments.